prusti-tests/tests/verify/fail/unsupported/type_param_array.rs

```diff
@@ -1,3 +1,5 @@
+// ignore-test slices of generics
+
```

Comment:
Does it crash now?

Reply:
Yes. As far as I can tell It's because the inhale generated because of the statement refers to `.val_ref` of the generic value, but that is not a thing in our generics. I think it would make things easier for us if we treated typarams as `Ref`s, just like ADTs.

Reply:
Does it panic or does it report an unsupported error?

Reply:
Right now it is a panic.

Reply:
I see.